src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java

```diff
@@ -79,202 +79,167 @@
 import org.wildfly.security.x500.X500;
 
 /**
- * Server-side authentication context.  Instances of this class are used to preform all authentication and re-authorization
+ * Server-side authentication context. Instances of this class are used to perform all authentication and re-authorization
```

Comment:
I'd really hate to lose the formatting on this JavaDoc.  I can't quite tell what the changes are.